src/averageAges.js

```diff
@@ -14,12 +19,17 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menAgeSum = people
+    .filter(person =>
+      century
+        ? Math.ceil(person.died / 100) === century
+        && person.sex === 'm'
+        : person.sex === 'm'
```

Comment:
if you doing something twice it is better to think for a helper variable or a function or do it in another way
```suggestion
        && isMan
        : isMan
```